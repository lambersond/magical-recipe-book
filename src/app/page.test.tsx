import { render } from '@test-utils'
import HomePage from './page'

describe('app/page', () => {
  it.skip('should render page', async () => {
    const { asFragment } = render(<HomePage />)
    expect(asFragment()).toMatchSnapshot()
  })
})
