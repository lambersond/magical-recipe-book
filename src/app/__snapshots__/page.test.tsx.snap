// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`app/page should render page 1`] = `
<DocumentFragment>
  <main
    class="pt-8 pb-20 sm:p-20 flex flex-col items-center gap-4"
  >
    <h1
      class="text-3xl font-bold"
    >
      Welcome to the Home Page
    </h1>
    <p
      class="text-lg"
    >
      This is a sample application demonstrating code splitting and lazy loading.
    </p>
  </main>
</DocumentFragment>
`;
